### ISSUE TITLE
RFE: date added?

### ISSUE THREAD

**Issue by joshmoore** [sbesson/idr-ome-ngff-samples#1]
This could likely be pulled from the generator metadata, but ordering the file by date added would help users to find the most recent additions. (We might also be able to auto-generate a changelog from that)

**Comment by sbesson** [comment]
Makes sense. While looking at https://joshmoore.github.io/idr-ome-ngff-samples/, one of the first UI actions I wanted to do was to reverse sort the OME-NGFF version to put the most recent samples to the top of the list so I can see the benefit of a date added column. I assume for historical samples, the timestamp of `.zattrs` on uk1s3 would be a good indication?

**Comment by sbesson** [comment]
Unfortunately, `curl -I https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.1/9798462.zarr/.zattrs` reveals we won't be able to get anything older than the date where the data was copied from the Embassy CLoud v3 object store to the Embassy Cloud v4 object store (uk1s3). Possibly, an alternative would be to use the blog post date for all 0.1, 0.2 and 0.3 samples and start capturing the date of generation of new sampels

**Comment by joshmoore** [comment]
Doh! I did not think of that as a requirement. 🤦🏽 The blog dates make sense (and I guess we could even think them as a "releaseUrl" or something)

**Comment by joshmoore** [comment]
Cheers. Can we discuss merging by branch into the mainline so things get automatically updated?

**Comment by sbesson** [comment]
Absolutely, do you want to push directly to the `main` branch (I think I sent a collaborator invite a week ago) or open a PR?

**Comment by joshmoore** [comment]
Done :+1:
